src/documentation/pages/components/skeleton/examples/_complex-combination.vue

```diff
@@ -0,0 +1,93 @@
+<template lang="pug">
+  <ComponentExample title="Complex combination" id="complex-combination" additionalClasses="-p-lg--6" :tabs="exampleTabs">
+    <Wrapper slot="example">
+      button.chi-button#toggle-skeleton Toggle Skeleton
+      .complex-combination-html.-d--none
+        .-text--h2 Lorem ipsum dolor sit amet
+        p.-mw--480
+          | Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          | eiusmod tempor incididunt ut labore et dolore magna aliqua.
+          | Ut enim ad minim veniam, quis nostrud exercitation.
+        .-d--flex.-mw--480.-mt--4
+          .chi-avatar
+            i.chi-icon.icon-user(aria-hidden="true")
+          .chi-form__item.-flex--grow1.-ml--2
+            input#example__placeholder.chi-input(type='text' placeholder='Placeholder' aria-label='Example input')
+          button.chi-button.-primary.-ml--2 Send
+      .complex-combination-skeleton
```

Comment:
```suggestion
      .complex-combination-skeleton(v-else)
```
I suggest you change the order as well, like putting the Skeleton first with the condition v-if="isSkeletonVisible" and then the complex HTML with v-else

Reply:
done now!